net.sourceforge.fuse_emulator.Fuse.appdata.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 The Fuse development team -->
+<component type="desktop">
+  <id>net.sourceforge.fuse_emulator.Fuse</id>
+  <provides>
+    <id>net.sf.fuse_emulator.desktop</id>
+  </provides>
+  <replaces>
+    <id>net.sf.fuse_emulator.desktop</id>
+  </replaces>
```

Comment:
it's either `provides` or `replaces`. In that case `replaces` is most appropriate.